pkg/cmd/release/shared/fetch.go

```diff
@@ -21,22 +21,23 @@ import (
 )
 
 var ReleaseFields = []string{
-	"url",
 	"apiUrl",
-	"uploadUrl",
-	"tarballUrl",
-	"zipballUrl",
-	"id",
-	"tagName",
-	"name",
+	"author",
```

Comment:
This is both an alphabetising and the addition of `databaseId`. You can see the individual commits for clarity.

Reply:
I'm always about alphabetizing, helps find duplicates